src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IncentiveLevelResourceTest.kt

```diff
@@ -28,17 +28,13 @@ class IncentiveLevelResourceTest : IncentiveLevelResourceTestBase() {
   }
 
   @Nested
-  inner class `retrieving incentive levels` {
```

Comment:
Would be better to group tests by the endpoint tested rather than RO/RW? Too granular?

Reply:
Ignore me, watching the whole diff with whitespaces just confused me 😅